.github/workflows/baselines/download_artifacts.sh

```diff
@@ -15,18 +15,20 @@ fi
 
 [ "$#" -ge "1" ] || usage
 
+EXPECTED_NUM_PAGES=2
```

Comment:
Is this safe to hard-code this value here?

Reply:
I think it's okay for now because the number of pages will only increase once the number of jobs surpasses 200, and we're only at 107 right now, but this could definitely lead to errors in the future. I will update this to compute the number of pages based on the artifact count